browsergym/experiments/src/browsergym/experiments/loop.py

```diff
@@ -34,15 +34,15 @@
 @dataclass
 class EnvArgs(DataClassJsonMixin):
     task_name: str
-    task_seed: int = None
-    max_steps: int = None
+    task_seed: Optional[int] = None
```

Comment:
what is the advantage of putting optional here?
(I don't oppose, but I never do it because I'm not sure when it's useful)

Reply:
DataClassJson was complaining (warning in the logs)